### ISSUE TITLE
Metricbeat fails to list all processes when run as a non-privileged user

### ISSUE THREAD

**Issue by adriansr** [elastic/beats#12301]
When running Metricbeat under Windows as a regular, non-administrator user, the process / process_summary metricsets won't report processes belonging to other users.

The log is filled with messages like
```
2019-05-24T13:29:59.487+0200 DEBUG [processes] process/process.go:460 Skip process pid=444: error getting process state for pid=444: getProcCredName failed: OpenProcessToken failed 
```

**Comment by adriansr** [comment]
@narph I assigned you both after discussing the issue with @odacremolbap 

I had a quick fix for this in #12305, however, this PR is causing metricbeat to complain with an error after new code paths are being reached in the Windows+non-admin-user case:

```
ERROR   instance/metrics.go:92  Error while getting memory usage: error retrieving process stats: cannot find matching process for pid=2456
```

This will make `tests/system/test_system.py:Test.test_process` to fail because it doesn't want any ERROR or WARN in the logs.

**Comment by narph** [comment]
@adriansr , https://github.com/elastic/beats/pull/12475 has been merged
